.github/workflows/linux-build-release.yml

```diff
@@ -38,6 +38,12 @@ jobs:
         run: ./gradlew functionalTest
       - name: Documentation tests
         run: ./gradlew docTest
+      - name: Upload test reports
```

Comment:
There's really no need to upload test results, as you can view them with the help of a build scan. The build scan link is available at the end of each workflow run, on success and on failure. I'd say remove this change.